app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,40 @@
+/* universal */
+
+body {
+padding-top: 60px;
```

wrong indent
Fix the same error in this file
```suggestion
  padding-top: 60px;
```